### PR TITLE
EOS-17486:Remove s3iamcli installation from s3server.install

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

**Review by AjinkyaDhumal — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
Looks good to me...

File: srv/components/s3server/install.sls [hunk 1]
```diff
@@ -31,9 +31,9 @@ Install s3server package:
     - version: {{ pillar['commons']['version']['cortx-s3server'] }}
     - refresh: True
 
-Install cortx-s3iamcli:
-  pkg.installed:
-    - pkgs:
-      - cortx-s3iamcli: {{ pillar['commons']['version']['cortx-s3iamcli'] }}
-      # - s3iamcli-devel
-      # - s3server-debuginfo
+#Install cortx-s3iamcli:
```
Comment (83bhp) [inline srv/components/s3server/install.sls:34]: Please delete the lines instead of commenting.

**Review by 83bhp — CHANGES_REQUESTED** [review]
Changes look ok, please also remove the s3iamcli from teardown state.

**Review by 83bhp — APPROVED** [review]
Looks good!

**Review by amitwac2608 — APPROVED** [review]

**Comment by nitin-seagate** [comment]
Merge this changes to main branch

**Pull request merged**